action.yml

```diff
@@ -52,16 +55,19 @@ inputs:
     required: true
     default: 'latest'
   headerOnly:
-    description: 'Whether Gobra should only verify files with the header "// +gobra"'
+    description: 'Whether Gobra should only verify files with the header "// +gobra".'
     required: false
     default: '0'
   module:
-    description: 'Name of current module that should be used for resolving imports'
+    description: 'Name of current module that should be used for resolving imports.'
+    required: false
+  assumeInjectivityOnInhale:
+    description: 'Assume injectivity of the receiver expression when inhaling quantified permissions, instead of checking it, like in Viper versions previous to 2022.02.'
     required: false
+    default: '0'
```

Comment:
This will become the default behaviour of Gobra eventually, so I think that it is a good idea to start using it already when possible. Users that are aware of this change and and to disable it still have an option though